test-prow-e2e-cypress.sh

```
@@ -0,0 +1,44 @@
+if [ ! -d node_modules ]; then
+  yarn install
+fi
+
+SCENARIO="${1:-e2e}"
+
+if [ "$SCENARIO" == "nightly-cypress" ]; then
+  PACKAGE=""
+  if [ $# -gt 1 ]; then
+    PACKAGE="-p $2"
+  fi
+  ./integration-tests/test-cypress.sh -n true $PACKAGE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)